src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -213,7 +213,7 @@ export default {
 </docs>
 
 <template>
-	<div class="rich-contenteditable">
+	<div class="rich-contenteditable" :class="class">
```

Comment:
`$props.class` should work fine, so it won't mix it with `class` keyword

Reply:
May we implement it also for master to avoid [styles breaking](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4907#issuecomment-1856039941)?

Reply:
In `master` in Vue 2 it works the same by default, because `class` and `style` attributes don't count as `$attrs` and are not managed by `inheritAttrs`. So in master setting `class` on `<NcRichContenteditable>` sets it on the root anyway.

The change in this PR resolves breaking change between `master` and `next`, moving the class back to root.